### ISSUE TITLE
Refactor index preparation

### ISSUE THREAD

**Issue by quetcodesfire** [stac-utils/stac-api#16]

**Comment by matthewhanson** [comment]
Index preparation (currently done here https://github.com/stac-utils/stac-api/blob/develop/libs/es.js#L80) should occur during a postinstall deployment phase, rather than having to check every time a new item is ingested.

This is not a fully published plugin: https://github.com/ec-europa/eubfr-data-lake/tree/master/plugins/serverless-plugin-elasticsearch-index but may have some ideas on how to do this.

Basically after deployment we want to create the indices and mappings

**Comment by matthewhanson** [comment]
Forked the plugin to fix some issues: https://github.com/matthewhanson/serverless-plugin-elasticsearch-index

Need to clean that up and publish to npm, but in the meantime stac-api is just installing from agithub tag: 1.0.0-b2

Pulled out index mappings from code and created fixtures so the indices with mappings is created at deploy time.
